layouts/page/nx-cugraph.html

```diff
@@ -0,0 +1,325 @@
+        </div>
+      </div>
+    </div>
+</section>
+
+
+<div class="top-slant-container">
+  <div class="top-slant-up bg-rapids-gray"></div>
+</div>
+
+<section class="row section bg-rapids-gray padding-bottom-slant">
+  <div class="col">
+    <div class="container">
+      
+      <h1 class="mt-10 mb-4"> How It Works </h1>
+
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-regular fa-light fa-gear-code"></i> Under the Hood </h2>
+          <p>NetworkX has the ability to dispatch function calls to separately-installed third-party backends. These NetworkX backends let users experience improved performance and/or additional functionality without changing their NetworkX Python code. Examples include backends that provide algorithm acceleration using GPUs, parallel processing, graph database integration, and more.</p>
+          
+          <br>
+          
+          <p>While NetworkX is a pure-Python implementation, backends may be written to use other libraries and even specialized hardware. As such, `nx-cugraph` is a NetworkX backend that uses RAPIDS cuGraph and NVIDIA GPUs to significantly improve NetworkX performance.</p>
+          
+          <br>
+          
+          In a simple script <code>demo.ipy</code>
+
+          <code class="code-sample mt-3">
+            import networkx as nx <br>
+            import pandas as pd <br>
+            <br>
+            url = "https://data.rapids.ai/cugraph/datasets/cit-Patents.csv" <br>
+            df = pd.read_csv(url, sep=" ", names=["src", "dst"], dtype="int32") <br>
+            G = nx.from_pandas_edgelist(df, source="src", target="dst") <br>
+            <br>
+            %time result = nx.betweenness_centrality(G, k=10)
+          </code>
+
+          <code class="code-sample mt-3">
+            user@machine:/# ipython demo.ipy<br>
+            <br>
+            CPU times: user 7min 36s, sys: 5.22 s, total: 7min 41s <br>
+            Wall time: 7min 41s <br>
+          </code>
+
+          <code class="code-sample mt-3">
+            user@machine:/# NX_CUGRAPH_AUTOCONFIG=True ipython demo.ipy<br>
+            <br>
+            CPU times: user 4.14 s, sys: 1.13 s, total: 5.27 s <br>
+            Wall time: 5.32 s<br>
+          </code>
+          <p class="fs-8 fw-lighter text-center">*NetworkX 3.4.1, nx-cugraph 24.10, CPU: Intel(R) Xeon(R) Gold 6128 CPU @ 3.40GHz 45GB RAM, GPU: NVIDIA Quadro RTX 8000 50GB RAM </p>
+        </div>
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-ballot-check"></i> Execution Flow </h2>
+          {{ with .Resources.GetMatch "nxcg-execution-diagram.jpg" }}
+          <img src="{{ .RelPermalink }}">
+          {{ end }}
+        </div>
+      </div>
+
+      <h1 class="mt-10 mb-4"> Try nx-cugraph Yourself </h1>
+      <p class="subheading  mb-5">Find where to get more resources and connect with developers.</p>
+
+      <div class="row">
```

Comment:
Should this go here to make the "Get Started" button work similar to cudf-pandas?
```suggestion
      <div class="row" id=get-started>
```

Reply:
Resolved in #419